ext/bin/pdbbox-init

```diff
@@ -176,4 +176,40 @@ ssl-key = $abs_pdbbox/ssl/pdb.key.pem
 disable-update-checking = true
 EOF
 
+# do some tuning on PostgreSQL
+
+# generally accepted configuration parameters:
+# Set the shared_buffers to be 1/4 of the amount of RAM
+
+# for development only (considered unsafe in production):
+# Turn off fsync
+# Turn off synchronous commit
+
+case "$OSTYPE" in
+    darwin*)
+        total_ram_bytes=$(sysctl hw.memsize | awk '{ print $2 }') 
```

Comment:
Not critical, but suspect we might be able to avoid the external awk via "${parameter##word}", etc. expansions.  Though not worth it if it gets too complicated...